Packs/SymantecEndpointProtection/Integrations/SymantecEndpointProtection_V2/SymantecEndpointProtection_V2.py

```diff
@@ -804,7 +800,7 @@ def move_client_to_group_command(token):
     Before EVERY command the following tow lines are performed (do_auth and get_token_from_response)
     '''
     resp = do_auth(server=demisto.getParam('server'), crads=demisto.getParam(
-        'authentication'), insecure=demisto.getParam('insecure'))
+        'authentication'), insecure=demisto.getParam('insecure'), domain=demisto.params().get('domain', ''))
```

Comment:
be consistent with the code:
```suggestion
        'authentication'), insecure=demisto.getParam('insecure'), domain=demisto.getParam('domain'))
```